app/assets/javascripts/actions/campaign_actions.js

```diff
@@ -93,9 +93,9 @@ const fetchAllCampaignsPromise = async () => {
   return response.json();
 };
 
-export const fetchAllCampaigns = () => (dispatch) => {
+export const fetchAllCampaigns = (userOnly, newest) => (dispatch) => {
```

Comment:
Are these both supposed to be true/false values? A comment on how to use the parameters would be helpful. Does the name `fetchAllCampaigns` still accurately describe the function, or should it be renamed?

Reply:
What  does `fetchCampaigns` do, and can it be renamed?

Reply:
I've renamed this to `fetchCampaignStatistics`. The `fetchAllCampaigns` function now returns all the campaigns(but without the statistics)